### ISSUE TITLE
Callouts from ITS Theme don't show up in Callout Suggestions

### ISSUE THREAD

**Issue by Pachtjuilaard** [cwfryer/obsidian-callout-suggestions#9]
The callouts in that theme (like !recite and !aside) can be found in Callout Manager, but don't show up using your plugin, Callout Suggestions. I've tried manually setting things like icon and colors in light and dark themes, but that didn't work.

**Comment by cwfryer** [comment]
There might be some other plugin interfering.
On a fresh install with just callout manager, callout suggestions, and ITS-Theme it looks like it is working fine:
![Screenshot from 2024-10-21 09-30-47](https://github.com/user-attachments/assets/b681547d-d132-456a-8e58-d4cd199dbae7)

**Comment by Pachtjuilaard** [comment]
Looks like I misunderstood the assignment and didn't check your screenshot well enough. Found that out after an hour of trial and error 🤦‍♂️ I did find that Admonitions has it's own suggester (which looks different as well), but that triggers when typing "> [!". I thought this plugin would show when typing the ! in the same place as well. Turns out I have to type ">!", or any other character that's been set as the completion trigger character.

Anyway, it's working! Thanks!
